test/libsolidity/semanticTests/events/event.sol

```diff
@@ -0,0 +1,31 @@
+contract ClientReceipt {
+    event Deposit(address indexed _from, bytes32 indexed _id, uint _value);
+    function deposit(bytes32 _id, bool _manually) public payable {
+        if (_manually) {
+            bytes32 s = 0x19dacbf83c5de6658e14cbf7bcae5c15eca2eedecf1c66fbca928e4d351bea0f;
+            uint value = msg.value;
+            address sender = msg.sender;
+            assembly {
+                mstore(0, value)
+                log3(0, 0x20, s, sender, _id)
+            }
+        } else {
+            emit Deposit(msg.sender, _id, msg.value);
+        }
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// deposit(bytes32,bool), 18 wei: 0x1234, true ->
+// - log[0]
+// -   topic[0]: keccak256('Deposit(address,bytes32,uint256)')
+// -   topic[1]: 0000000000000000000000001212121212121212121212121212120000000012 (address)
+// -   topic[2]: 0000000000000000000000000000000000000000000000000000000000001234 (bytes32)
+// -    data[0]: 0000000000000000000000000000000000000000000000000000000000000012 (uint256)
```

Comment:
This looks good already, but I would prefer this to be shortened to
```suggestion
// - emit Deposit(address,bytes32,uint256): 0x1212121212121212121212121212120000000012, 0x1234, 0x12
```

Maybe we can distinguish topics and data by e.g. prefixing something like `#` to the topic data.

Reply:
Ok, so you prefer to have everything in one line, where topics and data can be differentiated by using different prefixes?
e.g. 
```
// - emit Deposit(address,bytes32,uint256): 0x1212121212121212121212121212120000000012, 0x1234, #0x12
```

Reply:
We don't do it for functions, why should we do it for events? I think topics should have the prefix because they are usually not complex.